src/app/onion/learning-object-builder/pages/outcome-page/outcome-page.component.html

```diff
@@ -2,10 +2,9 @@
   <div left>
     <clark-scaffold *ngIf="learningObject" [learningObject]= "learningObject"></clark-scaffold>
   </div>
-  <div main>
+  <div main class="main">
     <div class="top">
-      <div class="title">Learning Outcomes</div>
-      <button id="form" class="button good" aria-label="Add New Outcome" (activate)="newOutcome()">New <i class="fas fa-plus"></i></button>
+      <div class="title" *ngIf = "iterableOutcomes.length !== 1; else singleOutcome">{{iterableOutcomes.length}} Learning Outcomes</div>
```

Comment:
You can also conditionally add an s in the span and you don't need to use the template